### PR TITLE
show current time and date on the clock tab.

### PR DESCRIPTION
**Pull request by YaroslavP1995**
Show Current time and date on the clock tab. PR my task .

### CODE REVIEW

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    <TextClock
+            android:id="@+id/text_date_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="16sp"
+            android:textColor="#786543"
+            android:format24Hour="“EEE, d MMM”"
+            android:format12Hour="“EEE, d MMM”"
+            android:timeZone="GMT+0300"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Why your timezone is here? You don't need to hardcode it. It need to take time from your system!

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    <TextClock
+            android:id="@+id/text_clock_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="33sp"
+            android:textColor="#29b6f6"
+            android:format24Hour="HH:mm:ss"
+            android:format12Hour="h:mm:ss"
+            android:timeZone="GMT+0300"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Why your timezone is here? You don't need to hardcode it. It need to take time from your system!

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+    <TextClock
+            android:id="@+id/text_clock_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="33sp"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: move to dimens

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            android:textColor="#29b6f6"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: move to colors

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            android:format24Hour="HH:mm:ss"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Move to strings

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+            android:format12Hour="h:mm:ss"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Move to strings

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+            android:layout_alignParentTop="true"
+    />
+
+    <TextClock
+            android:id="@+id/text_date_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="16sp"
+            android:textColor="#786543"
+            android:format24Hour="“EEE, d MMM”"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Move to strings

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+            android:format12Hour="“EEE, d MMM”"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Move to strings

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+            android:layout_centerInParent="true"
+            android:layout_alignParentTop="true"
+    />
+
+    <TextClock
+            android:id="@+id/text_date_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="16sp"
+            android:textColor="#786543"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Move to colors

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+            android:timeZone="GMT+0300"
+            android:layout_centerInParent="true"
+            android:layout_alignParentTop="true"
+    />
+
+    <TextClock
+            android:id="@+id/text_date_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="16sp"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Move to dimens

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 11]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: Why you have 2 different clock_fragment layouts? Remove other one, this move to layout folder

File: app/src/main/java/com/pysiak/simple/clock/ui/clock/ClockFragment.java [hunk 12]
```diff
@@ -7,8 +7,13 @@
 import android.view.LayoutInflater;
```
Comment (Artem-Zelinka) [inline app/src/main/java/com/pysiak/simple/clock/ui/clock/ClockFragment.java]: No any real change in this class. So remove all new imports and new lines

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+            android:id="@+id/text_clock_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="33sp"
+            android:textColor="#29b6f6"
+            android:format24Hour="HH:mm:ss"
+            android:format12Hour="h:mm:ss"
+            android:timeZone="GMT+0300"
+            android:layout_centerInParent="true"
+            android:layout_alignParentTop="true"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: You first clock need to be at the top of screen and centered horizontally. AlignParentTop is incorrect

File: app/src/main/res/layout-v17/clock_fragment.xml [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+    <TextClock
+            android:id="@+id/text_date_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="16sp"
+            android:textColor="#786543"
+            android:format24Hour="“EEE, d MMM”"
+            android:format12Hour="“EEE, d MMM”"
+            android:timeZone="GMT+0300"
+            android:layout_centerInParent="true"
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout-v17/clock_fragment.xml]: You second view need to be after the first view and need to be centered horizontally. AlignParentTop is incorrect

**Review by Artem-Zelinka — CHANGES_REQUESTED** [review]
Remove all unnecessary changes. Remove 1 of your layouts. Move all values to dimens/colors/strings. Fix first and second views placement

**Comment by YaroslavP1995** [comment]
Please rewiew my change .

File: app/src/main/res/layout/clock_fragment.xml [hunk 15]
```diff
@@ -1,13 +1,29 @@
...
             android:layout_height="wrap_content"
-            android:layout_centerInParent="true"
-            android:text="@string/clock_tab_title"
-            android:textSize="36sp" />
+            android:format12Hour="@string/clock_format_time_12_hour"
+            android:format24Hour="@string/clock_format_time_24_hour"
+            android:textSize="@dimen/clock_text_time_size"
+            android:textColor="@color/primaryViewColor"
+            android:gravity="center_horizontal"
+            tools:targetApi="jelly_bean_mr1"/>
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout/clock_fragment.xml]: What is this? Why you need this?

File: app/src/main/res/layout/clock_fragment.xml [hunk 16]
```diff
@@ -1,13 +1,29 @@
...
+    <TextClock
+            android:id="@+id/text_date_id"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:format12Hour="@string/clock_format_date_12_hour"
+            android:format24Hour="@string/clock_format_date_24_hour"
+            android:textSize="@dimen/clock_text_date_size"
+            android:textColor="@color/secondaryViewColor"
+            android:gravity="center_horizontal"
+            tools:targetApi="jelly_bean_mr1"/>
```
Comment (Artem-Zelinka) [inline app/src/main/res/layout/clock_fragment.xml]: What is this? Why you need this?

File: app/src/main/res/values/styles.xml [hunk 17]
```diff
@@ -3,8 +3,8 @@
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
         <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorPrimary">@color/secondaryViewColor</item>
```
Comment (Artem-Zelinka) [inline app/src/main/res/values/styles.xml]: Why you change it in style?

**Review by Artem-Zelinka — CHANGES_REQUESTED** [review]
Don't use anything you don't know how to use and for what reason... Don't change anything without any reason

**Review by Artem-Zelinka — APPROVED** [review]

**Comment by YaroslavP1995** [comment]
Turned the styles back

**Pull request merged**